### ISSUE TITLE
Gas station brands as car washes

### ISSUE THREAD

**Issue by 1ec5** [osmlab/name-suggestion-index#3059]
Some gas station brands like Shell and BP are also commonly applied to car washes. As with convenience stores, a particular location may use the gas station brand or a separate brand on its car wash. New car wash entries for Shell et al. can reuse the existing Wikidata and Wikipedia tags.

**Comment by 1ec5** [comment]
Some of these names are in the discard list:

https://github.com/osmlab/name-suggestion-index/blob/9372a71bea0dbbe4a392cf5dce82dbab500773a2/dist/names_discard.json#L257

**Comment by SilentSpike** [comment]
Are we talking about standalone car washes or car washes on the site of a petrol station?

If the latter, I don't tend to tag those with `name` since they're on site and not named individually. The station itself can be mapped as an area instead of a point (encompassing the car wash).

**Comment by 1ec5** [comment]
This issue is about car washes on the site of a gas station.

It may be a point of regional variation. In the U.S., at least in places I’m familiar with, such car washes may or may not be named, but regardless they are often branded the same way as the on-site convenience store (for which we already have entries), and people would expect to be able to search for the car wash by the brand name just as they would the convenience store.

**Comment by bhousel** [comment]
I just realized this issue has been open for a while.. 

We did have a `amenity/car_wash.json` file with a few car washes manually added to it, but we weren't pulling in car washes collected from the planet file.  In 031b8ffc0 I removed the `"skipCollection": true` setting from this file so the build script will now include the collected car washes. 

Now the file includes a lot of car washes that are cobranded with fuel stations, so I went through and matched these up with whatever we already have in `amenity/fuel.json`.
